src/vma/util/lock_wrapper.h

```diff
@@ -68,7 +68,7 @@ class lock_base
 #endif
 protected:
 	tscval_t                start_lock_wait() { return 0; };
-	void                    end_lock_wait(tscval_t start_time) {start_time = 0; NOT_IN_USE(start_time);};
+	void                    end_lock_wait(tscval_t start_time) {start_time = 0; start_time = start_time;};
```

Comment:
1. can you explain what is wrong with NOT_IN_USE()?
2. can we remove the 'start_time = 0;'